projects/packages/assets/src/class-script-data.php

```diff
@@ -215,6 +215,10 @@ protected static function get_current_user_data() {
 		return array(
 			'display_name' => $current_user->display_name,
 			'id'           => $current_user->ID,
+			'permissions'  => array(
+				'manage_options' => current_user_can( 'manage_options' ),
+				'manage_modules' => current_user_can( 'jetpack_manage_modules' ),
+			),
```

Comment:
Why not simply use this?

```suggestion
			'permissions'  => $current_user->allcaps,
```

Reply:
I don't think that will work for `jetpack_manage_modules` as it's a custom one handled with the map_meta_cap filter.

Reply:
I think `$user->caps` should contain those extra caps.